config/routes.rb

```diff
@@ -158,9 +158,11 @@
         as: :risc_configuration
 
     get '/account' => 'accounts#show'
+    get '/account/connected_accounts' => 'accounts#connected_accounts'
```

Comment:
The "Rails way" would be to make an `Accounts::ConnectedAccounts#show` controller, do we can stick to the "built in" actions like `index` `show` etc etc, WDYT of pulling this into a separate controller? Ditto for `#history` and `#two_factor_authentication`

Reply:
actually rethinking it after starting to do that. All the actions share a `before_action`, `layout`, and structure. I think it may make sense to keep them in the same controller with that?

Reply:
the `before_action` is a really common one, and specifying a layout is a one-liner, so I think it would be fine to duplicate? Also the history page and connected accounts do not need PII decrypted, right? So the view model could be created separately? Or maybe a reason to break up the view model, since it's no longer just AccountShow? (view model could be a separate follow-up PR if we wanted to keep this small-ish)

Reply:
Sounds good to me, I've split up the controllers.

I think it makes sense to break apart the AccountShow, but have removed the PII loading from the new pages that don't use it.